ingeniamotion/fsoe.py

```diff
@@ -262,6 +262,14 @@ def __init__(
         if not FSOE_MASTER_INSTALLED:
             return
         self.__servo = servo
+        self.__running = False
```

Comment:
there are many function calls on the constructor. Some expected this attribute to be present but was not ready yet. Moved all attributes here to be safer